### PR TITLE
Adds a console log to show js works

### PR DESCRIPTION
**Pull request by pdougall1**
This is just wiring up the `main.js` file with the html file

### CODE REVIEW

**Comment by pdougall1** [comment]
@akell47 Nice!  What do you think, can we get this js working!?

**Comment by akell47** [comment]
oh!

File: index.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
     <svg xmlns="http://www.w3.org/2000/svg">
       <circle cx="50" cy="50" r="50" fill="red" />
     </svg>
-    
+
+    <script type="text/javascript" src="scripts/main.js"></script>    
   </body>
 </html>
```
Comment (pdougall1) [inline index.html:16]: this little red arrow is saying I should add a new line to the file.  They should all have that at the end.  Something about C parsing files or something, it's weird, we should just do it.

File: scripts/main.js [hunk 2]
```diff
@@ -0,0 +1 @@
+console.log('I RAN')
```
Comment (pdougall1) [inline scripts/main.js:1]: Same here I guess

**Review by pdougall1 — COMMENTED** [review]
haha, you shouldn't merge it actually.  Lets let the one who created the PR merge it.  You're reviewing it really.

**Comment by pdougall1** [comment]
@akell47 how about, sense you merged this one but there are some things we can clean up.  Those comments are referring to a red arrow you can't see anymore (because it's already merged I guess) but lets address the problem anyway

Can you checkout a branch, add new lines to the end of all the files, and submit a PR for those changes?

**Pull request merged**